packages/hooks/useStreamQueryVariables.js

```diff
@@ -24,11 +25,11 @@ export default function useStreamQueryVariables ({
       : null,
     childPostInclusion: currentUser?.settings?.streamChildPosts || 'yes',
     collectionToFilterOut: null,
-    context: isContextGroupSlug(forGroup?.slug)
-      ? forGroup.slug
+    context: context || (isContextGroupSlug(forGroup?.slug) // Relying on context now but will fall back to prior logic
```

Comment:
This is likely to trip-up the currentGroup state, and I need to chat about this to understand if this is the only and best option and why.